src/lib/modules/debugger/index.ts

```diff
@@ -426,7 +437,7 @@ class TransactionDebugger {
       }
       this.embark.logger.info("vars:");
       foundVars.forEach((variable: any) => {
-        this.embark.logger.info(`${variable.name}: ${variable.value}`);
+        this.embark.logger.info(`${variable.name}: ` + `${variable.value}`.white);
```

Comment:
Might make sense to move this (or at least the three spots with the for loop) in a function, so we don't repeat ourselves here.

Reply:
Oh, well, same comment as Pascal then